src/main/resources/styles.yml

```
@@ -25,3 +25,8 @@ lore_weapon_damage:
 lore_weapon_ammo:
   key: "<!i><white>"
   ammo: "<yellow>"
+
+lore_ammo_count:
+  key: "<!i><white>"
+  count: "<green>"
+  max: "<blue>"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)